osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs

```diff
@@ -44,8 +30,8 @@ private void layout()
                     EdgeEffect = new EdgeEffect
                     {
                         Type = EdgeEffectType.Glow,
-                        Radius = 20,
-                        Colour = glowColour,
+                        Radius = 10,
+                        Colour = Color4.Gray.Opacity(0.2f),

```

Comment:
if it was using osucolour it's just going to be using the gray generation function. which i don't think will ever be skinnable